src/main/java/com/bunq/sdk/exception/ApiException.java

```diff
@@ -1,37 +1,34 @@
 package com.bunq.sdk.exception;
 
-import java.util.List;
-
-/**
- * Exception triggered by API requests failed on the server side.
- */
 public class ApiException extends RuntimeException {
 
-  private static final String GLUE_ERROR_MESSAGES = "\n";
-
+  /**
+   */
   private int responseCode;
-  private List<String> messages;
+  private String message;
 
   /**
-   * @param responseCode The HTTP Response code of the failed request.
-   * @param messages The list of messages related to this exception.
    */
-  public ApiException(int responseCode, List<String> messages) {
-    super(concatenateMessages(messages));
-    this.responseCode = responseCode;
-    this.messages = messages;
-  }
+  public ApiException(){super();}
```

Comment:
Why do we need this constructor at all? And if we need it, why is it one-lined?

Reply:
Ahhh stupid IDE. So I started coding the exceptions and the IDE was complaining that there was no default constructor: https://stackoverflow.com/questions/4488716/java-default-constructor.

So I created on to shut it up :D But now that there is one, it can go 😄.